File: app/src/main/java/com/sanousun/mdweather/ui/activity/AboutActivity.java
package com.sanousun.mdweather.ui.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.Menu;
import android.view.MenuItem;

import com.sanousun.mdweather.R;
import com.sanousun.mdweather.ui.fragment.AboutFragment;

import butterknife.Bind;
import butterknife.ButterKnife;

public class AboutActivity extends AppCompatActivity {

    @Bind(R.id.toolbar)
    Toolbar mToolbar;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        ButterKnife.bind(this);
        mToolbar.setTitle(R.string.about);
        setSupportActionBar(mToolbar);
        if (getSupportActionBar() != null)
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        getFragmentManager().
                beginTransaction().
                add(R.id.about_container, AboutFragment.newInstance()).
                commit();
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        if (item.getItemId() == android.R.id.home) {
            this.finish();
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/java/com/sanousun/mdweather/activity/AboutActivity.kt
package com.sanousun.mdweather.activity

import android.os.Bundle
import com.sanousun.mdweather.R
import com.sanousun.mdweather.ui.fragment.AboutFragment
import kotlinx.android.synthetic.main.include_toolbar.*

/**
 * Created by dashu on 2017/6/27.
 * 关于界面
 */

class AboutActivity : BaseActivity() {

    override fun setLayoutId(): Int {
        return R.layout.activity_about
    }

    override fun initData(savedInstanceState: Bundle?) {
        //null
    }

    override fun initView() {
        toolbar.title = "关于"
        setSupportActionBar(toolbar)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
        fragmentManager
                .beginTransaction()
                .add(R.id.about_container, AboutFragment.newInstance())
                .commit()
    }
}
